src/check_release.rs

```diff
@@ -64,6 +64,97 @@ fn classify_semver_version_change(
     }
 }
 
+/// Helper function to print an error message about a failed lint.
+fn print_failed_lint(
+    config: &mut GlobalConfig,
+    semver_query: &SemverQuery,
+    results: Vec<BTreeMap<Arc<str>, FieldValue>>,
```

Comment:
It's a bit unusual to take ownership of the value we're printing when we don't have to do that. The only printing aid that takes ownership is `dbg!()` but that's because it returns the value it received so it can be used as an expression. None of the other printing aids (`println!()` etc.) take ownership, so we probably shouldn't either.
```suggestion
    results: &[BTreeMap<Arc<str>, FieldValue>],
```